target/linux/ath79/image/nand.mk

```diff
@@ -1,13 +1,13 @@
 # attention: only zlib compression is allowed for the boot fs
 define Build/zyxel-buildkerneljffs
-	rm -rf  $(KDIR_TMP)/zyxelnbg6716
-	mkdir -p $(KDIR_TMP)/zyxelnbg6716/image/boot
-	cp $@ $(KDIR_TMP)/zyxelnbg6716/image/boot/vmlinux.lzma.uImage
+	rm -rf  $@.tmp
```

Comment:
It's not really usual to delete the directory at the beginning if you look at other commands.
Since it's device-specific, the only file there will be added by copying and we remove it afterwards, I think this line can be just dropped.

Reply:
Correct, I'll fix it.